src/color-chart/color-chart.js

```diff
@@ -75,6 +75,30 @@ const Self = class ColorChart extends ColorElement {
 
 	bounds = { x: { min: Infinity, max: -Infinity }, y: { min: Infinity, max: -Infinity } };
 
+	// Follow the provided min/max values, if any
+	get force () {
+		const ctx = this;
```

Comment:
What's the point of using `ctx` instead of `this`? It doesn't provide any performance improvement and is only one character shorter, so it doesn't seem like the indirection is worth it.